### PR TITLE
 [AFL] Prevent Radamsa and MLRNN from polluting corpus with oversized testcases

### LINKED ISSUE #828: Radamsa and ML RNN are adding testcases over 1 MB to AFL's corpus
This prevents AFL from running.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
Fixes #828

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
CC @mukundv-chrome

File: src/python/tests/core/bot/fuzzers/afl/afl_launcher_test.py [hunk 1]
```diff
@@ -710,8 +710,22 @@ def _initialization_for_run_afl(self):
...
         lambda *args, **kwargs: self._process_result())
 
+    self.runner.strategies.decide_fast_cal_random(
+        len(os.listdir(self.runner.afl_input.input_directory)))
     self._write_bad_input()
 
+  def test_do_offline_mutation_large_testcase(self):
+    """Tests that do_offline_mutation doesn't add oversized testcases to the
+    corpus."""
+    contents = 'A' * 2**20  # 1 MB
```
Comment (inferno-chromium) [inline src/python/tests/core/bot/fuzzers/afl/afl_launcher_test.py]: Can you add in test with smaller file that does continue to exist.

**Pull request merged**